packages/server/api/src/app/flows/flow/flow.controller.ts

```diff
@@ -290,6 +291,9 @@ async function validateTriggerType(
   const blockTrigger = flow.version.trigger;
 
   if (blockTrigger.type !== TriggerType.BLOCK) {
+    logger.warn(
```

Comment:
yes yes, this shouldnt be here but i dont open a new pr :pleading_face::point_right::point_left: